### PR TITLE
fix status LEDs for separate nozzle and logo LEDs

### PR DESCRIPTION
**Pull request by kdomanski**
The macro aways uses TRANSMIT=0 for the logo LEDs, which assumes that the subsequent call to nozzle LEDs will flush the data. That will only work if both sets of LEDs are on the same chain.

The chains can be different though - configured through `status_leds_nozzle_led_name` and `status_leds_logo_led_name`. With this change, separate logo LEDs will use TRANSMIT=1 instead.

### CODE REVIEW

**Comment by kdomanski** [comment]
Tested on my Voron v0.2, with logo LEDs connected to the main MCU and nozzle LEDs connected to toolhead MCU.

File: macros/hardware_functions/status_leds.cfg [hunk 1]
```diff
@@ -96,6 +96,9 @@ gcode:
 [gcode_macro STATUS_LEDS]
 gcode:
     {% set color = params.COLOR|default('off')|lower %}
+    {% set logo_leds_name = printer["gcode_macro _USER_VARIABLES"]["status_leds_logo_led_name"] %}
+    {% set nozzle_leds_name = printer["gcode_macro _USER_VARIABLES"]["status_leds_nozzle_led_name"] %}
+    {% set logo_transmit = 0 if (logs_leds_name == nozzle_leds_name) else 1 %}
```
Comment (Frix-x) [inline macros/hardware_functions/status_leds.cfg:101]: I think there is a mistake in the variable name. Can you fix it? I'll merge it when it's done :) 
```suggestion
    {% set logo_transmit = 0 if (logo_leds_name == nozzle_leds_name) else 1 %}
```

**Review by Frix-x — CHANGES_REQUESTED** [review]

**Comment by Frix-x** [comment]
Hey, thanks for the PR, it's a great patch! I'll merge it after you fix the typo :)

File: macros/hardware_functions/status_leds.cfg (hunk 1 above)
Comment (kdomanski) [reply macros/hardware_functions/status_leds.cfg]: Ah, nice catch. Fixed!

**Comment by Surion79** [comment]
Very nice! 👍🏻

**Review by Frix-x — APPROVED** [review]
Thanks, I'll merge it :)

**Pull request merged**